packages/@react-spectrum/overlays/src/Modal.tsx

```diff
@@ -95,15 +88,19 @@ let ModalWrapper = forwardRef(function (props: ModalWrapperProps, ref: RefObject
   };
 
   return (
-    <div className={wrapperClassName} style={style}>
-      <div
-        {...mergeProps(otherProps, overlayProps, modalProps)}
-        ref={ref}
-        className={modalClassName}
-        data-testid="modal">
-        {children}
+    <>
+      <Underlay {...underlayProps} isOpen={isOpen} />
+      <div className={wrapperClassName} style={style}>
+        <div
```

Comment:
weird that style props get applied to two different elements? the className goes on the outer one but style goes on the inner?

Reply:
It's not. Style props get applied below. The style on the wrapper div is not from props. This did not change from before.

Reply:
you are right, it did not change from before and github code folding fooled me that it was being split